src/sentry/integrations/github/search.py

```diff
@@ -9,12 +9,12 @@
 class GitHubSearchEndpoint(OrganizationEndpoint):
     permission_classes = (OrganizationPermission, )
 
-    def get(self, request, organization, integration_id):
+    def get(self, request, organization, provider_id, integration_id):
         try:
             integration = Integration.objects.get(
                 organizations=organization,
                 id=integration_id,
-                provider='github',
```

Comment:
Let's just get rid of the provider filter here, since it's coming from the URL anyway we're no longer really validating anything and can just trust that the integration_id is a github or github enterprise integration.